.github/workflows/python-poetry-release.yaml

```diff
@@ -57,6 +58,10 @@ on:
         description: "The old version of the package."
         value: ${{ jobs.create-release.outputs.old-version }}
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}-python-poetry-release
+  cancel-in-progress: true
+
```

Comment:
We do not want workflows that create tags or releases to every be canceled while they are running